workflow/lib/shared/illumination_correction.py

```diff
@@ -90,9 +85,9 @@ def calculate_ic_field(
     threading: bool = False,
     slicer: slice = slice(None),
 ) -> np.ndarray:
-    """
-    Calculate illumination correction field for use with the apply_illumination_correction
-    Snake method. Equivalent to CellProfiler's CorrectIlluminationCalculate module with
+    """Calculate illumination correction field for use with the apply_ic_field.
```

Comment:
I would maybe hyperlink these. This will become more of a thing during phenotyping:

https://github.com/CellProfiler/CellProfiler/blob/fa81fb0f2850c7c6d9cefdf4e71806188f1dc546/src/frontend/cellprofiler/modules/correctilluminationcalculate.py#L96

Reply:
We can't hyperlink in the docstrings but I did add both of these in https://github.com/cheeseman-lab/brieflow/pull/6/commits/17334ec5e081359f1a673eb9172fa24a96b085a8